fe/fe-core/src/main/java/org/apache/doris/analysis/SyncStmt.java

```diff
@@ -27,6 +27,6 @@ public void analyze(Analyzer analyzer) throws UserException {
 
     @Override
     public RedirectStatus getRedirectStatus() {
```

Comment:
why do you change this？

Reply:
Many users are using sync command to sync current fe meta with master.

Reply:
According to the discussion with @dataroaring , SyncStmt is better to reuse `syncJournal`, which sync meta equivalently to the previous way but a little bit lighter.